cmd/rollappd/cmd/root.go

```diff
@@ -109,6 +112,18 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 			//create Block Explorer Json-RPC toml config file
 			berpcconfig.EnsureRoot(home, berpcconfig.DefaultBeJsonRpcConfig())
 
+			// Set config
+			sdkconfig := sdk.GetConfig()
+			utils.SetBip44CoinType(sdkconfig)
+			cfg := serverCtx.Config
+			genFile := cfg.GenesisFile()
+			if tmos.FileExists(genFile) {
+				genDoc, _ := GenesisDocFromFile(genFile)
+				rdk_utils.SetPrefixes(sdkconfig, genDoc["bech32_prefix"].(string))
+			} else {
+				rdk_utils.SetPrefixes(sdkconfig, "ethm")
```

Comment:
@trinitys7 in what cases the `else` clause is needed?

Reply:
It is used when we don't have `bech32_prefix` field in genesis. Normally we will start initializing from `init` genesis then `bech32_prefix` will be written. But if we start with `keys add` or some similar command, some errors will appear. Of course this will just be the default case and it's fine in e2e tests: 
`failed to create faucet accounts: failed to create common account with name faucet: failed to get account address for key "faucet" on chain rollapp1-1: invalid Bech32 prefix; expected rol, got ethm`

Reply:
if there is no else clause the error may be `invalid Bech32 prefix; expected rol, got cosmos`

Reply:
I mean it's okay if we delete this clause, but it would be better for e2e-test to have it